doc/administrator_guide.md

```diff
@@ -240,4 +292,3 @@ To upgrade the dataset schema, do the following.
         kcidb-db-dump -d bigquery:kernelci01_archive > kernelci01_archive.json
         # Using new-schema kcidb
         kcidb-db-load -d bigquery:kernelci02 < kernelci01_archive.json
-
```

Comment:
Could you not remove this line? That would be unrelated to your actual change.

Reply:
okay